templates/sim_im_calc.sl.template

```diff
@@ -35,7 +35,7 @@ fi
 
 # Check that the result files exist
 res=0
-if [[ -f {{sim_dir}}/IM_calc/{{sim_name}}.csv ]] && [[ -f {{sim_dir}}/IM_calc/{{sim_name}}_im_calc.info ]]; then
+if [[ ! -f {{sim_dir}}/IM_calc/{{sim_name}}.csv ]] && [[ ! -f {{sim_dir}}/IM_calc/{{sim_name}}_im_calc.info ]]; then
```

Comment:
the naming is incorrect.
And I think its a good time to synchronize the expected output name 
(centralize the default output file name, check IM repo)